zerver/lib/push_notifications.py

```diff
@@ -561,7 +562,19 @@ def send_notifications_to_bouncer(
     assert isinstance(response_data["total_android_devices"], int)
     assert isinstance(response_data["total_apple_devices"], int)
 
-    return response_data["total_android_devices"], response_data["total_apple_devices"]
+    total_android_devices, total_apple_devices = (
+        response_data["total_android_devices"],
+        response_data["total_apple_devices"],
+    )
```

Comment:
Should we change the bouncer API to return `android_successfully_delivered + apple_successfully_delivered` as a separate parameter? I'm not sure there's really much utility to processing both values, though of course we'll need to keep returning them more or less indefinitely for backwards-compatibility.

Reply:
Yeah, these aren't very useful. But I think this is just #27483, where we noted

> the most meaningful extra information to add would be a list of any device IDs that are invalid and thus the server can garbage-collect its records related to.

So should I just note on that issue that the target design would probably be that list of device IDs to clean up + the actual number of successes?

Reply:
Yeah.